### PR TITLE
Update cosmos stake platform for Gaia fork

### PR DESCRIPTION
**Pull request by Pantani**
## Changes
- Move stake methods to `stake.go`
- Wrap the data inside the result field
- Fix unit tests

### CODE REVIEW

File: platform/cosmos/stake.go [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	for _, validator := range validators.Result {
+		results = append(results, normalizeValidator(validator, pool.Pool, inflationValue))
+	}
+
+	return results, nil
+}
+
+func (p *Platform) GetDetails() blockatlas.StakingDetails {
+	//TODO: Find a way to have a dynamic
+	return blockatlas.StakingDetails{
```
Comment (vikmeup) [inline platform/cosmos/stake.go]: Can we return max reward out of all validators

File: platform/cosmos/stake.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+	return results, nil
+}
+
+func (p *Platform) UndelegatedBalance(address string) (string, error) {
+	account, err := p.client.GetAccount(address)
+	if err != nil {
+		return "0", err
+	}
+	for _, coin := range account.Value.Coins {
+		if coin.Denom == "uatom" {
```
Comment (vikmeup) [inline platform/cosmos/stake.go]: uatom add to const

File: platform/cosmos/client.go [hunk 3]
```diff
@@ -105,15 +89,6 @@ func (c *Client) GetUnbondingDelegations(address string) (delegations []Unbondin
 
 func (c *Client) GetAccount(address string) (result Account, err error) {
 	path := fmt.Sprintf("auth/accounts/%s", address)
-	return result, c.Get(&result, path, nil)
-}
-
-var everstakeValidator = Validator{
```
Comment (vikmeup) [inline platform/cosmos/client.go]: Are you preserving this validator?

Comment (Pantani) [reply platform/cosmos/client.go]: now this is returned inside the validator endpoint

**Pull request merged**